docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -20,15 +21,18 @@ Reasons to use the package:
 1. Automatically enroll the identity and clean up the enrollment token in identity directory.
 1. Automatically upgrade the tunneller when a new package is available.
 
-Linux DEB packages are currently available for the x86_64 and arm64 platforms and RPM packages are available for x86_64. Additionally, there are executable downloads available for arm/v7 (32bit) for [manual installation](#manual-installation).
+Linux DEB packages are currently available for the x86_64 and arm64 platforms and RPM packages are available for x86_64.
+Additionally, there are executable downloads available for arm/v7 (32bit) for [manual
```

Comment:
I'd really ask you to favor omitting extraneous adverbs like Additionally. Restructure this to the more direct form:

> ARM/v7 (32bit) binaries are available via direct download from GitHub, see: [manual installation](#manual-installation).